src/core/tests/test_parallel_for_ordered.cc

```diff
@@ -0,0 +1,396 @@
+//------------------------------------------------------------------------------
+// Copyright 2019-2020 H2O.ai
```

Comment:
This seems to be a new file, so then may be `Copyright 2020`?

Reply:
one of the tests here existed in `ztest_parallel_for.cc`, which was split into several files (https://github.com/h2oai/datatable/pull/2557/files#diff-e59505a7fb028ead863ca29e3394e6f6)